faqs.html

```diff
@@ -182,12 +182,28 @@
 												<div class="tg-description">
 													<p><span>A.</span>- Full conference pass for two days.<br>
 													&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- Breakfast, Lunch and Hi-Tea.<br>
-													&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- PyCon India T-shirt which will be distributed on Sep 25.
+													&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- PyCon India T-shirt which will be distributed on November 5th.
```

Comment:
This is good @CuriousLearner We can make a change later if required, for now its good to go.